packages/core/react/src/__tests__/WalletProviderMobile-test.tsx

```diff
@@ -470,8 +474,8 @@ describe('WalletProvider when the environment is `MOBILE_WEB`', () => {
                             mobileWalletAdapter.disconnect();
                         });
                     });
-                    it('should not clear the stored wallet name', () => {
-                        expect(localStorage.removeItem).not.toHaveBeenCalled();
+                    it('should clear the stored wallet name', () => {
+                        expect(localStorage.removeItem).toHaveBeenCalled();
```

Comment:
Hmm. I think the original intent of the test is correct -- the wallet disconnecting should NOT deselect the wallet or remove it from LocalStorage, because then autoconnect will be broken on reload.

Reply:
is this true? AFAICT when using wallet adapter autoconnect is not supposed to reload the last used wallet if disconnect was called